src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -338,13 +348,115 @@ impl DragonballInner {
         Ok((old_vcpus, new_vcpus))
     }
 
+    pub(crate) fn resize_memory(
+        &mut self,
+        old_mem_mb: u32,
+        new_mem_mb: u32,
+    ) -> Result<(u32, MemoryConfig)> {
+        // check the invalid request memory
+        if new_mem_mb > self.hypervisor_config().memory_info.default_maxmemory
+            && self.hypervisor_config().memory_info.default_maxmemory > 0
```

Comment:
what about set it to intmax or host total memory when it is 0?

Reply:
this `new_mem_mb ` variable cannot be 0, becuase it is calculated by adding it from the hypervisor's default memory.

Reply:
Does `default_maxmemory` can be 0?

Reply:
fixed